docs/ros/api/mcu_api.mdx

```diff
@@ -11,17 +11,17 @@ toc_max_heading_level: 4
-| platform/mcu/status/stop  | [clearpath_platform_msgs/StopStatus](https://github.com/clearpathrobotics/clearpath_msgs/blob/main/clearpath_platform_msgs/msg/StopStatus.msg)       | MCU node              | Stop loop status                                       | [Sensor Data](overview.mdx#sensor-data)         |
-| platform/motors/cmd_drive | [clearpath_platform_msgs/Drive](https://github.com/clearpathrobotics/clearpath_msgs/blob/main/clearpath_platform_msgs/msg/Drive.msg)                 | Control node          | Command velocities to individual wheels                | [Sensor Data](overview.mdx#sensor-data)         |
-| platform/motors/feedback  | [clearpath_platform_msgs/DriveFeedback](https://github.com/clearpathrobotics/clearpath_msgs/blob/main/clearpath_platform_msgs/msg/DriveFeedback.msg) | MCU node              | Feedback from motor drive unit                         | [Sensor Data](overview.mdx#sensor-data)         |
+| platform/mcu/_cmd_lights  | [clearpath_platform_msgs/Lights](https://github.com/clearpathrobotics/clearpath_msgs/blob/humble/clearpath_platform_msgs/msg/Lights.msg)               | Lighting command      | Hidden topic used to send lighting commands to the MCU | [Sensor Data](overview.mdx#sensor-data)         |
+| platform/mcu/status       | [clearpath_platform_msgs/Status](https://github.com/clearpathrobotics/clearpath_msgs/blob/humble/clearpath_platform_msgs/msg/Status.msg)               | MCU node              | MCU status                                             | [Sensor Data](overview.mdx#sensor-data)         |
+| platform/mcu/status/power | [clearpath_platform_msgs/Power](https://github.com/clearpathrobotics/clearpath_msgs/blob/humble/clearpath_platform_msgs/msg/Power.msg)                 | MCU node              | Robot Power status                                     | [Sensor Data](overview.mdx#sensor-data)         |
+| platform/mcu/status/stop  | [clearpath_platform_msgs/StopStatus](https://github.com/clearpathrobotics/clearpath_msgs/blob/humble/clearpath_platform_msgs/msg/StopStatus.msg)       | MCU node              | Stop loop status                                       | [Sensor Data](overview.mdx#sensor-data)         |
+| platform/motors/cmd_drive | [clearpath_platform_msgs/Drive](https://github.com/clearpathrobotics/clearpath_msgs/blob/humble/clearpath_platform_msgs/msg/Drive.msg)                 | Control node          | Command velocities to individual wheels                | [Sensor Data](overview.mdx#sensor-data)         |
+| platform/motors/feedback  | [clearpath_platform_msgs/DriveFeedback](https://github.com/clearpathrobotics/clearpath_msgs/blob/humble/clearpath_platform_msgs/msg/DriveFeedback.msg) | MCU node              | Feedback from motor drive unit                         | [Sensor Data](overview.mdx#sensor-data)         |
```

Comment:
Can you confirm that you want the Jazzy docs to point to the humble branch instead of the jazzy branch?

Reply:
I do want the Jazzy branch, thanks for catching that.